pip/compat.py

```diff
@@ -41,7 +46,14 @@ def console_to_str(s):
         try:
             return s.decode(sys.__stdout__.encoding)
         except UnicodeDecodeError:
-            return s.decode('utf_8')
+            if WINDOWS:
+                try:
+                    from ctypes import cdll
+                    return s.decode("cp" + str(cdll.kernel32.GetACP()))
```

Comment:
Yeah, I have no problem with that. You might get the wrong characters, but it's not likely to do any harm.

The only possible issue (and this occurs if we try UTF-8 and it works, as well) is that the output contains some characters which are decodable using whatever encoding we end up succeeding with, but which aren't encodable in ```sys.stdout.encoding```. But that's likely to be very rare in practice (and will go away in Python 3.6, which uses Unicode for console output, and so can print anything).